packages/medusa-plugin-sanity/__mocks__/contentful-management.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
```

Comment:
This file should be .gitignored

Reply:
https://vuuzletvph.com/what-is-the-secret-of-nft-tokens-and-why-they-are-bought-for-millions-of-dollars/

Kindly check this out.Have a great day ahead!

Reply:
https://vuuzletvph.com/heads-up-vuco-bounty-programs-are-coming-your-way/